websites/T/Twitch/presence.ts

```diff
@@ -102,10 +102,10 @@ const statics = {
 presence.on("UpdateData", async () => {
   const path = location.pathname.replace(/\/?$/, "/");
 
-  const showBrowsing = await presence.getSetting("browse");
-  const showLive = await presence.getSetting("live");
-  const showVideo = await presence.getSetting("video");
-  const showTimestamps = await presence.getSetting("timestamp");
+  const showBrowsing = true,
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)